Still some issues with tests:
```
[INFO] Running org.eclipse.kura.internal.json.marshaller.unmarshaller.test.JsonEncoderDecoderTest
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'org.apache.logging.log4j.simplelog.StatusLogger.level' to TRACE to show Log4j2 internal initialization logging.
[ERROR] Tests run: 28, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.609 s <<< FAILURE! - in org.eclipse.kura.internal.json.marshaller.unmarshaller.test.JsonEncoderDecoderTest
[ERROR] testSystemDeploymentPackages(org.eclipse.kura.internal.json.marshaller.unmarshaller.test.JsonEncoderDecoderTest)  Time elapsed: 0.008 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<{"[packages":[{"name":"dp1","version":"1.0.0","bundles":[{"name":"bundle1","version":"2.0.0]"}]}]}> but was:<{"[deploymentPackages":[{"name":"dp1","version":"1.0.0","bundles":[{"name":"bundle1","version":"2.0.0","id":0,"state":"ACTIVE]"}]}]}>
	at org.eclipse.kura.internal.json.marshaller.unmarshaller.test.JsonEncoderDecoderTest.testSystemDeploymentPackages(JsonEncoderDecoderTest.java:502)

[INFO] Running org.eclipse.kura.internal.json.marshaller.unmarshaller.message.test.CloudPayloadJsonDecoderTest
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.002 s - in org.eclipse.kura.internal.json.marshaller.unmarshaller.message.test.CloudPayloadJsonDecoderTest
[INFO] Running org.eclipse.kura.internal.json.marshaller.unmarshaller.message.test.CloudPayloadJsonEncoderTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.002 s - in org.eclipse.kura.internal.json.marshaller.unmarshaller.message.test.CloudPayloadJsonEncoderTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   JsonEncoderDecoderTest.testSystemDeploymentPackages:502 expected:<{"[packages":[{"name":"dp1","version":"1.0.0","bundles":[{"name":"bundle1","version":"2.0.0]"}]}]}> but was:<{"[deploymentPackages":[{"name":"dp1","version":"1.0.0","bundles":[{"name":"bundle1","version":"2.0.0","id":0,"state":"ACTIVE]"}]}]}>
[INFO]
[ERROR] Tests run: 41, Failures: 1, Errors: 0, Skipped: 0
```

--- a/kura/test/org.eclipse.kura.json.marshaller.unmarshaller.provider.test/src/test/java/org/eclipse/kura/internal/json/marshaller/unmarshaller/test/JsonEncoderDecoderTest.java
+++ b/kura/test/org.eclipse.kura.json.marshaller.unmarshaller.provider.test/src/test/java/org/eclipse/kura/internal/json/marshaller/unmarshaller/test/JsonEncoderDecoderTest.java
@@ -498,7 +498,7 @@ public class JsonEncoderDecoderTest {
 
         String json = new JsonMarshallUnmarshallImpl().marshal(systemDeploymentPackages);
 
-        String expectedJson = "{\"packages\":[{\"name\":\"dp1\",\"version\":\"1.0.0\",\"bundles\":[{\"name\":\"bundle1\",\"version\":\"2.0.0\"}]}]}";
+        String expectedJson = "{\"deploymentPackages\":[{\"name\":\"dp1\",\"version\":\"1.0.0\",\"bundles\":[{\"name\":\"bundle1\",\"version\":\"2.0.0\",\"id\":0,\"state\":\"ACTIVE\"}]}]}";
         assertEquals(expectedJson, json);
     }
 

